ci-operator/config/integr8ly/heimdall/integr8ly-heimdall-master.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: '7'
+
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+  unit:
+    limits:
+      memory: 10Gi
+    requests:
+      memory: 2Gi
+      cpu: 200m
+
+images:
+- from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/integr8ly/heimdall/build
+  dockerfile_path: build/Dockerfile
+  to: heimdall
+
+build_root:
+  image_stream_tag:
+    name: heimdall-base-image
+    namespace: integr8ly
+    tag: latest
+
+binary_build_commands: make code/compile COMPILE_TARGET="./build/_output/bin/heimdall"
+
+promotion:
+  namespace: integr8ly
+  name: heimdall
+
+tag_specification:
+  name: '4.2'
+  namespace: ocp
+
+tests:
```

Comment:
Not sure if these are all the tests we want. I've looked at the other integreatly projects to see what they're doing there. There may be others we want here.

Reply:
We want format,unit and e2e for now https://github.com/openshift/release/blob/master/ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml#L48-L59

You need make sure all the targets exist in the Makefie first though https://github.com/integr8ly/heimdall/blob/master/Makefile

Reply:
format unit and e2e added. Resolving.